pkg/cloudcore/apis/config/types.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import commonconfig "github.com/kubeedge/kubeedge/pkg/common/apis/config"
+
+type CloudCoreConfig struct {
+	Kube             *KubeConfig             `json:"kube,omitempty"`
```

Comment:
```suggestion
	KubeAPIConfig             *KubeAPIConfig             `json:"kubeAPIConfig,omitempty"`
```

Reply:
The unversioned Config APIs should align with versioned APIs

Reply:
unversiond component api config is useless,  i has deleted it